weave/trace/weave_client.py

```diff
@@ -1022,61 +1150,31 @@ def _save_object_basic(
             ref = OpRef(self.entity, self.project, name, response.digest)
         else:
             ref = ObjectRef(self.entity, self.project, name, response.digest)
-        # TODO: Try to put a ref onto val? Or should user code use a style like
-        # save instead?
+
+        # Attach the ref to the object
```

Comment:
This already happened in all cases where we called this function. Adding here is the correct thing to do and simplified other code

Reply:
nice